react/index.js

```diff
@@ -280,7 +281,11 @@ export default class Footer extends Component {
                 </p>
               ))}
           </div>
-          <FooterVtexLogo logoUrl={logo} />
+          <RenderContextConsumer>
```

Comment:
Use `withRuntimeContext` instead. It provides the _account_ :)

Reply:
In the `FooterVtexLogo`

Reply:
I still prefer the use of Render Props, they provide a better understanding of what the component is receiving.
But if the use of the HOC is a definition, it's ok either.